phpcs.xml.dist

```diff
@@ -8,7 +8,7 @@
 	<rule ref="WordPress-Docs"/>
 	<rule ref="WordPress-Extra" />
 	<rule ref="WordPress.WP.I18n"/>
-	<config name="text_domain" value="performance-lab,default,auto-sizes"/>
+	<config name="text_domain" value="performance-lab,default,auto-sizes,dominant-color-images,webp-uploads"/>
```

Comment:
Note: Needs to be updated once https://github.com/WordPress/performance/pull/1002 is merged.